css/common.css

```diff
@@ -0,0 +1,254 @@
+         --font-quicksand: 'Quicksand', sans-serif;
+         /* ### Fonts Size */
+         --font-size--14px: 0.875rem;
+         --font-size--15px: 0.9375rem;
+         --font-size--16px: 1rem;
+         --font-size--17px: 1.0625rem;
+         --font-size--19px: 1.1875rem;
+         --font-size--20px: 1.25rem;
+         --font-size--21px: 1.3125rem;
+         --font-size--40px: 2.5rem;
+         /* ### Fonts Weight */
+         --font-weight--400: 400;
+         --font-weight--600: 600;
+         --font-weight--700: 700;
+     }
+     
+     body {
+         line-height: 1.5rem;
+         font-family: var(--font-quicksand);
+         font-size: var(--font-size--16px);
+         font-weight: var(--font-weight--400);
+     }
+     
+     @media screen and (min-width: 481px) {}
+     
+     @media screen and (min-width: 950px) {
+         nav {
+             display: unset;
+         }
+         .main__grid {
+             display: flex;
+             justify-content: center;
+             display: grid;
+             grid-template-columns: repeat(12, 1fr);
+             padding-top: 128px;
+         }
+         /* HEADER */
+         .header__desktop {
+             display: flex;
+         }
+         .header__desktop {
+             font-family: var(--font-poppins);
+             font-weight: var(--font-weight--700);
+             font-size: 1.6875rem;
+         }
+         .header__square--desktop {
+             background-color: var(--clr-royal-blue);
+             height: 16px;
+             width: 16px;
+             background-color: var(--clr-royal-blue);
+             margin-top: 4px;
+         }
+         .header__name--desktop {
+             padding-left: .8125rem;
+             font-family: var(--font-poppins);
+             font-size: var(--font-size--21px);
+             font-weight: var(--font-weight--700);
+         }
+         .header__role--desktop {
+             font-family: var(--font-quicksand);
+             font-size: var(--font-size--17px);
+             font-weight: var(--font-weight--400);
+             font-style: bold;
+             letter-spacing: 0.02em;
+         }
+         .header {
+             position: fixed;
+             grid-column: 1/-1;
+             height: 100%;
+             background-color: var(--clr-white);
+             height: 128px;
+             width: 100%;
+         }
+         .header__col2 a {
+             display: unset;
+         }
+         /* Profile */
+         .profile__role {
+             display: unset;
+         }
+         .profile {
+             grid-column: 1/7;
+             justify-content: flex-end;
+             background: linear-gradient(to right, #E6DACD 87%, #FFF 32%);
+             padding-bottom: 133px;
+             margin-right: 76px;
+             padding-top: 147px;
+         }
+         .profile__card {
+             width: 375px;
+             height: 524px;
+             right: 200px;
+             left: 212px;
+         }
+         .profile__card--img {
```

Comment:
well done for trying out BEM naming conventions! I think `img` and `name` are elements rather than modifiers, so these classes could be `profile__img` and `profile__name`